src/ceph-volume/ceph_volume/devices/lvm/zap.py

```diff
@@ -168,7 +168,8 @@ def zap_lv(self, device):
         """
         lv = api.get_single_lv(filters={'lv_name': device.lv_name, 'vg_name':
                                         device.vg_name})
-        pv = api.get_single_pv(filters={'lv_uuid': lv.lv_uuid})
+        #pv = api.get_single_pv(filters={'lv_uuid': lv.lv_uuid})
+        pvs = api.get_pvs(fields='vg_name,pv_name,pv_tags', filters={'vg_name': device.vg_name})
```

Comment:
is there some additional change happening here in respect to having multiple pvs? This doesn't seem tied to the addition of the `--no-destroy` flag